README.md

```diff
@@ -118,6 +118,7 @@ Compared to ChatGLM's [P-Tuning](https://github.com/THUDM/ChatGLM2-6B/tree/main/
 | [XVERSE](https://huggingface.co/xverse)                  | 7B/13B/65B                  | q_proj,v_proj     | xverse    |
 | [Yi](https://huggingface.co/01-ai)                       | 6B/34B                      | q_proj,v_proj     | yi        |
 | [Yuan](https://huggingface.co/IEITYuan)                  | 2B/51B/102B                 | q_proj,v_proj     | yuan      |
+| [Lamma2-Chinese](https://huggingface.co/FlagAlpha)       | 7B/13B                      | q_proj,v_proj     | atom      |
```

Comment:
This line could be removed because the atom model shares the same architecture with llama2

Reply:
> This line could be removed because the atom model shares the same architecture with llama2

sure, i will remove this line.